File: Wirtualny_Swiat_Java/Projekt_PO_Java/src/model/Organisms/Grass.java
package model.Organisms;

import model.Organism;
import model.Plant;
import model.Point;
import model.World;

import java.awt.*;

public class Grass extends Plant {
    private final Color grassColor = Color.BLACK;

    public Grass(World world, Point position){
        super(world, position, 0);
    }

    @Override
    public Color getColor(){
        return grassColor;
    }

    @Override
    public String getNazwa(){
        return "Trawa";
    }

    @Override
    public Organism makeNewOrganism(Point position){
        return new Grass(world, position);
    }
}

File: WirtualnySwiatPython/Plants/Grass.py
from plant import Plant
from config import GREEN

class Grass(Plant):
    def __init__(self, world, position):
        super().__init__(world, position, power=0)
        self.color = GREEN

    def getColor(self):
        return self.color

    def getNazwa(self):
        return "Trawa"

    def makeNewOrganism(self, position):
        return Grass(self.world, position)
